mora/auth/auth.py

```diff
@@ -45,8 +46,8 @@ def __init__(self, assertion=None):
     def __call__(self, r):
         if self.assertion:
             assertion = self.assertion
-        elif flask.request:
-            assertion = flask.request.cookies.get(COOKIE_NAME)
+        elif flask.session and flask.session.get(COOKIE_NAME):
+            assertion = flask.session[COOKIE_NAME]
```

Comment:
You can simplify this code to:

```python
        elif flask.session:
             assertion = flask.session.get(COOKIE_NAME)
```

There's no need to avoid dealing with a `None` value, only to use `None` instead…

Also, we should rename the variable `COOKIE_NAME` to something like `TOKEN_KEY` now that it no longer names an actual cookie 🙂